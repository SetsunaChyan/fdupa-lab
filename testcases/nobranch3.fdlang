function main(){
x = 1;

x = x + x;
x = x + x;
x = x + x;
x = x + x;
x = x + x;
x = x + x;

check_interval(x, 64, 64);
check_interval(x, 128, 255);
check_interval(x, 1, 64);
}