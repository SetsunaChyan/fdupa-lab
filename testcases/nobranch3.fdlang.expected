Line 11: YES; x in [64, 64]
Line 12:  NO; x in [64, 64]
Line 13: YES; x in [64, 64]
